editor/editor_log.cpp

```diff
@@ -438,6 +529,9 @@ EditorLog::EditorLog() {
 	current = Thread::get_caller_id();
 
 	EditorNode::get_undo_redo()->set_commit_notify_callback(_undo_redo_cbk, this);
+
+	// Initialize RegEx for resource path detection
+	resource_path_regex.compile("\\bres:\\/\\/[^.]+[.]\\S+");
```

Comment:
```suggestion
	resource_path_regex.compile(R"(\bres://[^.]+[.]\S+)");
```

* Using raw string literal can make it easier to read.
* No need to escape forward slash.

Reply:
Ah, that's what that R before Strings I've been seeing in the source code means. I'm new to C++, Thanks!